apps/keyboard/js/layouts/emoji.js

```diff
@@ -313,52 +387,56 @@ Keyboards.emoji = {
-                 { compositeKey: '⛲', type: 'emoji' }, { compositeKey: '🎢', type: 'emoji' },
-                 { compositeKey: '🚢', type: 'emoji' }, { compositeKey: '⛵', type: 'emoji' },
-                 { compositeKey: '🚤', type: 'emoji' }, { compositeKey: '🚣', type: 'emoji' },
-                 { compositeKey: '⚓', type: 'emoji' }, { compositeKey: '🚀', type: 'emoji' },
-                 { compositeKey: '✈', type: 'emoji' }, { compositeKey: '💺', type: 'emoji' },
-                 { compositeKey: '🚁', type: 'emoji' }, { compositeKey: '🚂', type: 'emoji' },
-                 { compositeKey: '🚊', type: 'emoji' }, { compositeKey: '🚉', type: 'emoji' },
-                 { compositeKey: '🚞', type: 'emoji' }, { compositeKey: '🚆', type: 'emoji' },
-                 { compositeKey: '🚄', type: 'emoji' }, { compositeKey: '🚅', type: 'emoji' },
-                 { compositeKey: '🚈', type: 'emoji' }, { compositeKey: '🚇', type: 'emoji' },
-                 { compositeKey: '🚝', type: 'emoji' }, { compositeKey: '🚋', type: 'emoji' },
-                 { compositeKey: '🚃', type: 'emoji' }, { compositeKey: '🚎', type: 'emoji' },
-                 { compositeKey: '🚌', type: 'emoji' }, { compositeKey: '🚍', type: 'emoji' },
-                 { compositeKey: '🚙', type: 'emoji' }, { compositeKey: '🚘', type: 'emoji' },
-                 { compositeKey: '🚗', type: 'emoji' }, { compositeKey: '🚕', type: 'emoji' },
-                 { compositeKey: '🚖', type: 'emoji' }, { compositeKey: '🚛', type: 'emoji' },
-                 { compositeKey: '🚚', type: 'emoji' }, { compositeKey: '🚨', type: 'emoji' },
-                 { compositeKey: '🚓', type: 'emoji' }, { compositeKey: '🚔', type: 'emoji' },
-                 { compositeKey: '🚒', type: 'emoji' }, { compositeKey: '🚑', type: 'emoji' },
-                 { compositeKey: '🚐', type: 'emoji' }, { compositeKey: '🚲', type: 'emoji' },
-                 { compositeKey: '🚡', type: 'emoji' }, { compositeKey: '🚟', type: 'emoji' },
-                 { compositeKey: '🚠', type: 'emoji' }, { compositeKey: '🚜', type: 'emoji' },
-                 { compositeKey: '💈', type: 'emoji' }, { compositeKey: '🚏', type: 'emoji' },
-                 { compositeKey: '🎫', type: 'emoji' }, { compositeKey: '🚦', type: 'emoji' },
-                 { compositeKey: '🚥', type: 'emoji' }, { compositeKey: '⚠', type: 'emoji' },
-                 { compositeKey: '🚧', type: 'emoji' }, { compositeKey: '🔰', type: 'emoji' },
-                 { compositeKey: '⛽', type: 'emoji' }, { compositeKey: '🏮', type: 'emoji' },
-                 { compositeKey: '🎰', type: 'emoji' }, { compositeKey: '♨', type: 'emoji' },
-                 { compositeKey: '🗿', type: 'emoji' }, { compositeKey: '🎪', type: 'emoji' },
-                 { compositeKey: '🎭', type: 'emoji' }, { compositeKey: '📍', type: 'emoji' },
-                 { compositeKey: '🚩', type: 'emoji' }
+    panelKeys: [{ compositeKey: '🇨', type: 'emoji' }, { compositeKey: '🇪', type: 'emoji' },
```

Comment:
The flags are not encoded on it's right place. The U+1F1E8 along does not represent the flag of China -- the sequence of U+1F1E8 and U+1F1F3 would as shown below:

U+1F1E8 U+1F1F3
> 🇨🇳

U+1F1E8
> 🇨

U+1F1F3
> 🇳